proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -610,6 +632,96 @@ message AWSCloudTrailEventResource {
   string type = 2;
 }
 
+// KubeAuditLogStreamRequest is sent from the client (Teleport) to the server
+// (access graph) over the bi-directional KubeAuditLogStream. It encapsulates
+// distinct client actions for sending batches of Kubernetes audit log events
+// and managing the persistent resume state maintained by the access graph
+// server.
+message KubeAuditLogStreamRequest {
+  oneof action {
+    KubeAuditLogConfig config = 1;
+    KubeAuditLogEvents events = 2;
+  }
+}
+
+// KubeAuditLogConfig is the KubeAuditLogStream configuration. It is sent from
+// the client (Teleport) to the server (access graph) on start up, containing
+// any configuration parameters from the Teleport config. The server will
+// respond with an adjusted config that is that it will use, possibly
+// overriding the supplied config.
+message KubeAuditLogConfig {}
```

Comment:
This will likely be extended to include the log source for the stream, so that the resume states sent back include only that log source, and perhaps also a duration or start time for new clusters which specifies how far back to collect logs from for the initial sync.